server/CHANGELOG.md

```diff
@@ -1,12 +1,10 @@
-# Change Log
+# v 0.2.1 (?)
```

Comment:
0.2.1 is already released:
https://pypi.org/project/inmantals/

Reply:
I see. Does this mean I should update `setup.py` to contain 0.2.2?

Reply:
Our other release pipelines bump the version on master to the next minor release. To be consistent, it should be 0.3.0.

Reply:
Right, we added new features anyway, my reasoning was a bit off.